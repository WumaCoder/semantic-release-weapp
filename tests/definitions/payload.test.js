const payload = require('../../src/definitions/payload');

describe('payload', () => {
  test('payload matches snapshot of errors', () => {
    expect(payload('message')).toMatchSnapshot();
  });
});
