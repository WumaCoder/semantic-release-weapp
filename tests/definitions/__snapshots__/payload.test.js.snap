// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`payload payload matches snapshot of errors 1`] = `
Object {
  "body": "{\\"msg_type\\":\\"interactive\\",\\"card\\":{\\"elements\\":[{\\"tag\\":\\"div\\",\\"text\\":{\\"tag\\":\\"lark_md\\",\\"content\\":\\"message\\"}}]}}",
  "headers": Object {
    "Content-Type": "application/json",
  },
  "method": "POST",
}
`;
